pkg/delete/internal/classic/delete.go

```diff
@@ -38,33 +41,77 @@ type deleteValue struct {
 func Delete(ctx context.Context, client dtclient.Client, theApi api.API, entries []pointer.DeletePointer, targetApi string) error {
 	logger := log.WithCtxFields(ctx).WithFields(field.Type(theApi.ID))
 
-	values, err := client.ListConfigs(ctx, theApi)
-	if err != nil {
-		logger.WithFields(field.Error(err)).Error("Failed to fetch existing configs of API type %q - skipping deletion: %v", theApi.ID, err)
-		return err
-	}
-
 	deleteErrs := 0
```

Comment:
is the deleteErr counting actually correct? I think nit is increased too often and is not representing the amount of config items that could net be deleted... 🤔